packages/assets-controllers/src/NftController.ts

```diff
@@ -418,6 +423,10 @@ export class NftController extends BaseController<
     openSeaEnabled,
     isIpfsGatewayEnabled,
   }: PreferencesState) {
+    if (selectedAddress !== this.#selectedAddress) {
+      this.#lastNftRefreshTime = 0;
```

Comment:
Should the last time also be reset if the network changes?

Reply:
Removed in the last update 🙏